package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rise-common-electron",
-  "version": "2.2.1",
+  "version": "2.3.1",
```

Comment:
@jungeunyoon This isn't really a new feature and it isn't a breaking change (shouldn't be). Version can bump to `2.2.2`.